apps/nsqlookupd/nsqlookupd.go

```diff
@@ -20,8 +20,10 @@ func nsqlookupdFlagSet(opts *nsqlookupd.Options) *flag.FlagSet {
 
 	flagSet.String("config", "", "path to config file")
 	flagSet.Bool("version", false, "print version string")
-	flagSet.Bool("verbose", false, "enable verbose logging")
+
+	flagSet.Int("log-level", nsqlookupd.LOG_WARN, "set log verbosity: 1 through 5 (Debug, Info, Warn, Error, Fatal)")
```

Comment:
It would be. Could process the string into an int once the flag parsing is done, deal with errors, etc.

Is it worth the extra effort?